mods/ra/maps/soviet-06a/map.yaml

```diff
@@ -2,7 +2,7 @@ MapFormat: 11
 
 RequiresMod: ra
 
-Title: 06a: Bridge over the River Grotzny
+Title: 06a: Bridge Over The River Grotzny
```

Comment:
If we end up using English article conventions `over` would also be uncapitalized.